src/expr/impl/src/scalar/array_contain.rs

```diff
@@ -25,29 +25,35 @@ use risingwave_expr::function;
 ///
 /// ```slt
 /// query I
-/// select array[1,2,3] @> array[2,3];
+/// select array_contains(array[1,2,3], array[2,3]);
```

Comment:
Why do we switch to rw-only `array_contains` rather than pg-compatible `@>`?

Reply:
This file name confused me because I saw it was a documentation test placed inside the class. After a while I remembered it was an end-to-end test. 😕